lib/activerecord-clean-db-structure/clean_dump.rb

```diff
@@ -30,9 +30,9 @@ def run
       # Reduce noise for id fields by making them SERIAL instead of integer+sequence stuff
       #
       # This is a bit optimistic, but works as long as you don't have an id field thats not a sequence/uuid
-      dump.gsub!(/^    id integer NOT NULL,$/, '    id SERIAL PRIMARY KEY,')
```

Comment:
@lfittl Sure. The comma will need to be optional in the replacement text too, otherwise the comma-less last line will have a comma added to it. I've amended the commit to do this.